.github/workflows/ci.yml

```diff
@@ -153,6 +153,9 @@ jobs:
       - name: 🔎 Lint
         run: yarn lint
 
+      - name: 🌡 Test Types
+        run: yarn test:types --target 4.8,latest
```

Comment:
Can’t find what is the latest TypeScript version supported by RedWood. Trying out 4.8, easy to adjust.